### ISSUE TITLE
[gulp 4] It's no possible to create relative symlink 

### ISSUE THREAD

**Issue by vladkens** [gulpjs/gulp#1708]
Hi, i wanna create a relative symlink to the folder, but i always have a absolute.
It's a possible to create relative symlink? If possible please add example to the doc.

```
gulp.task('assets', function() {
    return gulp.src(['assets'])
        .pipe(gulp.symlink('dist'));
});

> gulp assets
> ll dist
lrwxr-xr-x  1 vladkens  staff    52B Jun 30 11:03 assets -> /Users/vladkens/xxx/assets

Can be:
assets -> ../assets
```

**Comment by phated** [comment]
This has also been brought up at https://github.com/gulpjs/vinyl-fs/issues/177 - it has never been supported.

**Comment by vladkens** [comment]
I close this issue because problem can be resolve by this commit https://github.com/gulpjs/vinyl-fs/commit/b76accc788d480e02cdbcb2a8a6b95ebeb2e3167

**Comment by phated** [comment]
Thanks @vladkens - That fix will be available in the vinyl-fs 3.0 release (which should be available pretty soon).
